packages/uikit/src/widgets/RoiCalculator/RoiCalculator.tsx

```diff
@@ -56,6 +56,7 @@ export type RoiCalculatorProps = {
     maxPrice: number;
     minPrice: number;
     averagePrice: number;
+    isLoading: boolean;
```

Comment:
let's change the prop name to `isPriceLoading` to prevent confusion

Reply:
sure